packages/dev-tools-pages/assets/index.html

```diff
@@ -1,14 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <link rel="preload" href="/fonts/MaisonNeue-Book-subset.woff2" as="font" type="font/woff2" crossorigin>
-  <link rel="preload" href="/fonts/MaisonNeue-Bold-subset.woff2" as="font" type="font/woff2" crossorigin>
-  <title><%= htmlWebpackPlugin.options.title %></title>
-</head>
-<body>
-  <div id="app"></div>
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="preload" href="/fonts/MaisonNeue-Book-subset.woff2" as="font" type="font/woff2" crossorigin />
+        <link rel="preload" href="/fonts/MaisonNeue-Bold-subset.woff2" as="font" type="font/woff2" crossorigin />
+        <title><%= htmlWebpackPlugin.options.title %></title>
+    </head>
+    <body>
+        <!-- Heap SDK -->
+        <script type="text/javascript">
+            (window.heap = window.heap || []),
+                (heap.load = function(e, t) {
+                    (window.heap.appid = e), (window.heap.config = t = t || {});
+                    var r = t.forceSSL || 'https:' === document.location.protocol,
+                        a = document.createElement('script');
+                    (a.type = 'text/javascript'),
+                        (a.async = !0),
+                        (a.src = (r ? 'https:' : 'http:') + '//cdn.heapanalytics.com/js/heap-' + e + '.js');
+                    var n = document.getElementsByTagName('script')[0];
+                    n.parentNode.insertBefore(a, n);
+                    for (
+                        var o = function(e) {
+                                return function() {
+                                    heap.push([e].concat(Array.prototype.slice.call(arguments, 0)));
+                                };
+                            },
+                            p = [
+                                'addEventProperties',
+                                'addUserProperties',
+                                'clearEventProperties',
+                                'identify',
+                                'resetIdentity',
+                                'removeEventProperty',
+                                'setEventProperties',
+                                'track',
+                                'unsetEventProperty',
+                            ],
+                            c = 0;
+                        c < p.length;
+                        c++
+                    )
+                        heap[p[c]] = o(p[c]);
+                });
+            heap.load('<%= htmlWebpackPlugin.options.heapAnalyticsId %>');
+        </script>
```

Comment:
You may not need this, but for instant we call `addEventProperties` with the `GIT_SHA` so we know what revision of the code the user is on